firedrake/slate/slate.py

```diff
@@ -567,7 +568,7 @@ class Action(TensorOp):
     :arg function: a :class:`firedrake.Function` object.
     """
 
-    prec = 2
+    prec = 3
```

Comment:
Pretty-printing and parenthesizing an expression when the C++ code-snippet is generated